dymos/trajectory/phase_linkage_comp.py

```diff
@@ -125,4 +134,7 @@ def compute(self, inputs, outputs):
             idxs_b = lnk._idxs_b
             output = lnk._output
 
-            outputs[output] = lnk['sign_a'] * inputs[input_a][idxs_a] + lnk['sign_b'] * inputs[input_b][idxs_b]
+            a_val = lnk['sign_a'] * (lnk._conv_a * inputs[input_a][idxs_a] + lnk._offset_a)
+            b_val = lnk['sign_b'] * (lnk._conv_b * inputs[input_b][idxs_b] + lnk._offset_b)
+
+            outputs[output] = a_val + b_val
```

Comment:
scale, offset = om.unit_conversion('degC', 'degF')
100.0 * scale + offset
197.77777777777771
(100.0 + offset)  * scale
211.99999999999991

So, these should be:

a_val = lnk['sign_a'] * (inputs[input_a][idxs_a] + lnk._offset_a) * lnk._conv_a